scripts/build-changelog.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+/*
+* This script generates changelog based on the json files in "../.changes/nextrelease/"
+* and generates a new TAG for the build.
+*/
+
+$dir = "../.changes/";
+$newServiceFlag = False;
+
+
+function readChangelog($dir){
+	$dir = $GLOBALS['dir']."/nextrelease/";
+	$CHANGELOG=[];
+	if (is_dir($dir)){
+	  if ($dh = opendir($dir)){
+	  	//Ignore any files starting with a (.) dot 
+	  	$files=preg_grep('/^([^.])/', scandir($dir));
+	  	foreach ($files as $file ){
+	      $str = file_get_contents($dir."/".$file);
+	      $CHANGELOG= array_merge($CHANGELOG, cleanJSON(json_decode($str)) );
+	    }
+	    closedir($dh);
+	  }
+	}
+	return $CHANGELOG;
+}
+
+function cleanJSON($arr){
```

Comment:
The indentation, bracket placement, and spacing should follow the [PSR-2](http://www.php-fig.org/psr/psr-2/) standard.